### ISSUE TITLE
zcash-cli walletpassphrase command takes passphrase as argument in clear text

### ISSUE THREAD

**Issue by hhanh00** [zcash/zcash#4932]
## Is your feature request related to a problem? Please describe.
The passphrase provided to the command is in clear and is captured by shell history and potentially clipboard tool

## Describe the solution you'd like
I'd like the ability to provide the passphrase after the command is issued and without echo

## Alternatives you've considered
I need to delete the shell history but I don't think it is a good solution in terms of security.

**Comment by str4d** [comment]
You can do this already with the `-stdin` argument for `zcash-cli`:

```
$ zcash-cli -stdin walletpassphrase\n
passphrase\n
<Ctrl+D>
$
```

**Comment by daira** [comment]
This almost, but doesn't quite satisfy the requested functionality, because the passphrase is echoed.

**Comment by str4d** [comment]
CLosed by #4936.
